pkg/command/proxy.go

```diff
@@ -0,0 +1,45 @@
+package command
+
+import (
+	"github.com/micro/cli/v2"
+	"github.com/owncloud/ocis-proxy/pkg/command"
+	svcconfig "github.com/owncloud/ocis-proxy/pkg/config"
+	"github.com/owncloud/ocis-proxy/pkg/flagset"
+	"github.com/owncloud/ocis/pkg/config"
+	"github.com/owncloud/ocis/pkg/register"
+)
+
+// ProxyCommand is the entry point for the proxy command.
+func ProxyCommand(cfg *config.Config) *cli.Command {
+	return &cli.Command{
+		Name:     "proxy",
+		Usage:    "Start proxy server",
+		Category: "Extensions",
+		Flags:    flagset.ServerWithConfig(cfg.Proxy),
+		Action: func(c *cli.Context) error {
+			serverConfig := configureProxy(cfg)
+			serverCommand := command.Server(serverConfig)
+
+			if err := serverCommand.Before(c); err != nil {
+				return err
+			}
+
+			return cli.HandleAction(serverCommand.Action, c)
+		},
+	}
+}
+
+func configureProxy(cfg *config.Config) *svcconfig.Config {
+	cfg.Proxy.Log.Level = cfg.Log.Level
+	cfg.Proxy.Log.Pretty = cfg.Log.Pretty
+	cfg.Proxy.Log.Color = cfg.Log.Color
+	cfg.Proxy.Tracing.Enabled = false
+	cfg.Proxy.HTTP.Addr = "localhost:9200"
+	cfg.Proxy.HTTP.Root = "/"
```

Comment:
remove these ... they are the defaults taken from ocis-proxy:
- tracing enabled default is not set (which equals to false): https://github.com/owncloud/ocis-proxy/blob/master/pkg/flagset/flagset.go#L58-L63
- http addr defaults to 0.0.0.0:9200 https://github.com/owncloud/ocis-proxy/blob/master/pkg/flagset/flagset.go#L120
  we can debate if we want to allow requests from everywhere or only from localhost. if we want to only allow localhost wen need to set the http address for all other services. then we should normalize across all ocis-commands
- http root is / by default: https://github.com/owncloud/ocis-proxy/blob/master/pkg/flagset/flagset.go#L127

Reply:
I vote for locking all services but ocis-proxy to localhost. that way only one service is reachable and ocis can be set up for testing in eg docker.